### DISCUSSION TITLE
Disable NPCs

### DISCUSSION THREAD

**Discussion by curif** [curif/AgeOfJoy-2022.1#29] [General]
A way to disable NPCs
Possible options:

* disable them, the gallery would then appear to be empty.
* stay static in it's origin.

I will open an issue for the movement.

**Comment by DavidJosephClarke** [comment]
A mid-way solution would be for them to get up and walk away whenever you get close, so you can see them in the background, but never interrupting you or nearby.

That does give me the funny idea: if you end up playing a game for a long time, you should ideally have NPCs start to congregate around you to watch, just like they would in real life. And when you stop playing they should all wander off again, ahaha.

**Comment by curif** [comment]
Issue #27 related to the first part of the suggestion. It's a bug.

The second, it's a good one!
